docs/how_to/cluster_hard_way.md

```diff
@@ -1,7 +1,8 @@
 # M3DB Cluster Deployment, Manually (The Hard Way)
 
 ## Introduction
-This document lists the manual steps involved in deploying a M3DB cluster. In practice, you wouldn’t be doing this by hand, you’d be automating it using Terraform or Kubernetes. If you’re interested in how you could do so, we have some other guides you might be interested under the How-To section on https://m3db.github.io/m3db/.
+
+This document lists the manual steps involved in deploying a M3DB cluster. In practice, you wouldn’t be doing this by hand, you’d be automating it using Terraform or Kubernetes. If you’re interested in how you could do so, we have some other guides you might be interested under the How-To section.
```

Comment:
Might read better as:

`This document lists the manual steps involved in deploying a M3DB cluster. In practice, you'd be automating this using Terraform or Kubernetes rather than doing this by hand; guides for doing so are available under the How-To section.`

Reply:
Sure thing, thanks.